hnn_core/dipole.py

```diff
@@ -106,20 +106,26 @@ def simulate_dipole(net, tstop, dt=0.025, n_trials=None, record_vsec=False,
     return dpls
 
 
-def _read_dipole_txt(fname):
+def _read_dipole_txt(fname, extension='.txt'):
     """Read dipole values from a txt file and create a Dipole instance.
 
     Parameters
     ----------
     fname : str
-        Full path to the input file (.txt)
-
+        Full path to the input file (.txt or .csv)
+    fname: io.StringIO
+        Content of file in memory as a StringIO
```

Comment:
I think it's better convention to combine these in the docstring. 

```suggestion
    fname: str or io.StringIO
        Full path to the input file (.txt or .csv) or content of file in memory as a StringIO
```

Reply:
Done